qiskit_machine_learning/kernels/fidelity_statevector_kernel.py

```diff
@@ -65,14 +84,21 @@ def __init__(
             cache_size: Maximum size of the statevector cache. When ``None`` this is unbounded.
             auto_clear_cache: Determines whether the statevector cache is retained when
                 :meth:`evaluate` is called. The cache is automatically cleared by default.
+            shots: The number of shots. If ``None``, the exact fidelity is used. Otherwise, the
+                mean is taken of samples drawn from a binomial distribution with probability equal
+                to the exact fidelity.
+            enforce_psd: Project to the closest positive semidefinite matrix if ``x = y``.
+                This is only required if the number of shots is specified.
```

Comment:
Since enforce_psd has a default value, can we say more like  `This is only used when number of shots given is not None.`